examples/covid19-question-1/README.md

```diff
@@ -47,38 +49,39 @@ This example model takes 13 features that include age, clinical symptoms and vit
     ```bash
     $ tar xvf synthetic_data.tar.gz
     x synthetic_data/
-    x synthetic_data/procedure_occurrence.csv
-    x synthetic_data/location.csv
-    x synthetic_data/visit_occurrence.csv
-    x synthetic_data/condition_era.csv
-    x synthetic_data/device_exposure.csv
-    x synthetic_data/drug_era.csv
-    x synthetic_data/observation.csv
-    x synthetic_data/goldstandard.csv
-    x synthetic_data/drug_exposure.csv
-    x synthetic_data/condition_occurrence.csv
-    x synthetic_data/person.csv
-    x synthetic_data/measurement.csv
-    x synthetic_data/observation_period.csv
+    x synthetic_data/training
+    x synthetic_data/evaluation
+    ```
+
+4. Create an `output` , `model` and `scratch` (optional) folders.
+
+    ```bash
+    mkdir output scratch model
     ```
 
-4. Create an `output` and `scratch` (optional) folders.
+5. Run the dockerized model to train on the patients in the training dataset.
 
     ```bash
-    mkdir output scratch
+    docker run \
+        -v $(pwd)/synthetic_data/training:/data:ro \
+        -v $(pwd)/output:/output:rw \
+        -v $(pwd)/scratch:/scratch:rw \
+        -v $(pwd)/model:/model:rw \
+        awesome-covid19-q1-model:v1 bash /app/train.sh
     ```
 
-5. Run the dockerized model to generate predictions for the patients included in the synthetic dataset.
+6. Run the trained model on evaluation dataset and generate the prediction file.
 
     ```bash
     docker run \
-        -v $(pwd)/synthetic_data:/data:ro \
+        -v $(pwd)/synthetic_data/evaluation:/data:ro \
         -v $(pwd)/output:/output:rw \
         -v $(pwd)/scratch:/scratch:rw \
```

Comment:
Pending on decision to remove scratch for inference.

Reply:
Should I keep it for now?

Reply:
Yes, an empty folder `scratch` will be available when running the script `infer.sh`.